### DISCUSSION TITLE
[SOLVED]

### DISCUSSION THREAD

**Discussion by francisluz** [cardano-foundation/cardano-graphql#618] [General]

**Comment by CyberCyclone** [comment]
Hey mate,

I've been using cardano-graphql in for ADATools.io for over a year. It's backed by cardano-do-sync which is used by Yoroi. Like all projects it can have unknown bugs but overall it's been incredibly stable.

**Reply by francisluz** [reply]
Hi buddy,

I open it in the wrong repo, for some reason it supposed to be on the JS-SDK. And yes graphql is pretty stable as I know too. Thanks for reply I'll close this discussion.
